### PR TITLE
Add Analog Theater scraper

### LINKED ISSUE #8: Need Analog Theater scraper

### PR DESCRIPTION
**Pull request by rattboi**
Fixes #8

### CODE REVIEW

File: trello/plugins/analogtheater.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    final_artists = []
+    for artist in artists:
+        artist = re.sub(r'\([^)]*\)', '', artist)
+        artist = artist.replace('w/', ',')
+        artist = artist.replace('/', ',')
+        artist = artist.replace('&', ',')
+        artist = artist.replace('featuring', ',')
+        artist = ','.join(artist.rsplit(' and ', 1))
+        artist = artist.split(',')
+        artist = map(string.strip, artist)
```
Comment (demophoon) [inline trello/plugins/analogtheater.py]: Strip is a method and not an attribute ;)

Comment (demophoon) [reply trello/plugins/analogtheater.py]: oh, wait, this is doing a map, ignore me.

Comment (demophoon) [reply trello/plugins/analogtheater.py]: Lets replace this with an anonymous function instead:
```python
artist = map(lambda x: x.strip(), artist)
```

This is for python 3 compatibility as string.strip doesn't exist there.

**Comment by rattboi** [comment]
Changes made

**Review by demophoon — APPROVED** [review]

**Pull request merged**